spec/support/mockers/mocker.rb

```diff
@@ -0,0 +1,69 @@
+require 'rspec/mocks/standalone'
+
+class Mocker < RSpec::Mocks::InstanceVerifyingDouble
```

Comment:
I'll definitely add some documentation.

I looked into other mocking frameworks, but I felt that what I wanted wasn't complex enough to warrant such a big change, considering that I had already written/updated a ton of tests to make heavy use of rspec-mocks. I also thought about just using `FactoryGirl.build_stubbed`, but I wanted to force a bit more explicit stubbing to encourage cleaner code and avoid callbacks. 

Also, the set of behavior I wanted to add to rspec's doubles is relatively small; I don't anticipate having to add much more to the base class other than a few more association stubbers.